### DISCUSSION TITLE
Disabling auto formatting

### DISCUSSION THREAD

**Discussion by quizhorniw** [mfussenegger/nvim-jdtls#699] [Q&A]
Hi there! Wondering if there's a way to disable auto formatting via particular command or similar? It would be great if it worked that way: auto formatting is enabled by default (this is excellent for my own projects), and when I open one of my team's project, I manually disable it while project is already loaded (I hope you've got the idea). Thank you!

**Comment by mfussenegger** [comment]
I'm not sure what you're referring to. nvim-jdtls has no auto-formatting ?

**Comment by brendensuarez** [comment]
relevant:
- https://www.reddit.com/r/spacemacs/comments/mzlul7/java_does_anyone_know_if_its_possible_to_tune/
- https://www.lazyvim.org/configuration/tips#disable-autoformat-for-some-buffers
